src/components/BottomSheet/BottomSheetContainer/BottomSheetContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {memo, useCallback, useMemo, useRef} from 'react'
+import {LayoutChangeEvent, StyleProp, View, ViewStyle} from 'react-native'
+import styled from 'styled-components'
+import {styles} from './styles'
+import type {BottomSheetContainerProps} from './types'
+
+const Container = styled(View)(() => ({
+  ...styles.container,
+}))
+
+const BottomSheetContainerComponent = ({
+  // containerHeight,
+  // containerOffset,
+  topInset = 0,
+  bottomInset = 0,
+  shouldCalculateHeight = true,
+  detached,
+  style,
+  children,
+}: BottomSheetContainerProps) => {
```

Comment:
Please change to use this config instead because we already agreed on it
`const Component: React.FC<ComponentProps> = ({}) => {}`